src/core_plugins/timelion/public/lib/_tests_/calculate_interval.js

```diff
@@ -0,0 +1,44 @@
+const filename = require('path').basename(__filename);
+const fn = require(`../calculate_interval`);
```

Comment:
pls rename to calculate_interval (had to scroll back up to realize what the fn does)